### PR TITLE
Add missing docs for FFI ctors

### PR DESCRIPTION
**Pull request by Manishearth**

### CODE REVIEW

File: ffi/capi/src/properties_maps.rs [hunk 1]
```diff
@@ -391,6 +408,7 @@ pub mod ffi {
                 self.0.as_borrowed().get_set_for_value(value),
             ))
         }
+        /// Create a map for the `Script` property, using compiled data.
```
Comment (sffc) [inline ffi/capi/src/properties_maps.rs:412]: Observation: Funny whitespace. (here and elsewhere)

File: ffi/capi/src/exemplar_chars.rs [hunk 2]
```diff
@@ -185,6 +200,7 @@ pub mod ffi {
                     .static_to_owned(),
             )))
         }
+        /// Create an [`ExemplarCharacters`] for the "main" set of exemplar characters for a given locale, using compiled data.
```
Comment (sffc) [inline ffi/capi/src/exemplar_chars.rs:204]: Whitespace is still funny

File: ffi/capi/src/properties_names.rs [hunk 3]
```diff
@@ -117,6 +123,7 @@ pub mod ffi {
                     .erase(),
             ))
         }
+        /// Create a name-to-enum mapper for the `East_Asian_Width` property, using a particular data source.
```
Comment (sffc) [inline ffi/capi/src/properties_names.rs:127]: Whitespace is still funny

File: ffi/capi/src/properties_names.rs [hunk 4]
```diff
@@ -142,6 +150,7 @@ pub mod ffi {
                     .erase(),
             ))
         }
+        /// Create a name-to-enum mapper for the `Bidi_Class` property, using a particular data source.
```
Comment (sffc) [inline ffi/capi/src/properties_names.rs:154]: Whitespace is still funny

File: ffi/capi/src/properties_names.rs [hunk 5]
```diff
@@ -157,12 +166,14 @@ pub mod ffi {
...
             )))
         }
+        /// Create a name-to-enum mapper for the `Indic_Syllabic_Category` property, using compiled data.
         #[diplomat::rust_link(icu_properties::props::IndicSyllabicCategory, Struct)]
         #[diplomat::attr(auto, named_constructor = "indic_syllabic_category")]
         #[cfg(feature = "compiled_data")]
         pub fn create_indic_syllabic_category() -> Box<PropertyValueNameToEnumMapper> {
             Box::new(PropertyValueNameToEnumMapper(icu_properties::PropertyParser::<icu_properties::props::IndicSyllabicCategory>::new().static_to_owned().erase()))
         }
+        /// Create a name-to-enum mapper for the `Indic_Syllabic_Category` property, using a particular data source.
```
Comment (sffc) [inline ffi/capi/src/properties_names.rs:177]: Whitespace is still funny

File: ffi/capi/src/properties_names.rs [hunk 6]
```diff
@@ -188,6 +200,7 @@ pub mod ffi {
                     .erase(),
             ))
         }
+        /// Create a name-to-enum mapper for the `Line_Break` property, using a particular data source.
```
Comment (sffc) [inline ffi/capi/src/properties_names.rs:204]: Whitespace is still funny

File: ffi/capi/src/properties_names.rs [hunk 7]
```diff
@@ -214,6 +228,7 @@ pub mod ffi {
                 .erase(),
             ))
         }
+        /// Create a name-to-enum mapper for the `Grapheme_Cluster_Break` property, using a particular data source.
```
Comment (sffc) [inline ffi/capi/src/properties_names.rs:232]: Whitespace is still funny

File: ffi/capi/src/properties_names.rs [hunk 8]
```diff
@@ -239,6 +255,7 @@ pub mod ffi {
                     .erase(),
             ))
         }
+        /// Create a name-to-enum mapper for the `Word_Break` property, using a particular data source.
```
Comment (sffc) [inline ffi/capi/src/properties_names.rs:259]: Whitespace is still funny

File: ffi/capi/src/properties_names.rs [hunk 9]
```diff
@@ -264,6 +282,7 @@ pub mod ffi {
                     .erase(),
             ))
         }
+        /// Create a name-to-enum mapper for the `Sentence_Break` property, using a particular data source.
```
Comment (sffc) [inline ffi/capi/src/properties_names.rs:286]: Whitespace is still funny

File: ffi/capi/src/properties_names.rs [hunk 10]
```diff
@@ -289,6 +309,7 @@ pub mod ffi {
                     .erase(),
             ))
         }
+        /// Create a name-to-enum mapper for the `Script` property, using a particular data source.
```
Comment (sffc) [inline ffi/capi/src/properties_names.rs:313]: Whitespace is still funny

File: ffi/capi/src/properties_unisets.rs [hunk 11]
```diff
@@ -49,6 +50,7 @@ pub mod ffi {
                 icu_properties::EmojiSetData::new::<BasicEmoji>().static_to_owned(),
             ))
         }
+        /// Create a map for the `Basic_Emoji` property, using a particular data source.
```
Comment (sffc) [inline ffi/capi/src/properties_unisets.rs:54]: Whitespace is still funny

File: ffi/capi/src/script.rs [hunk 12]
```diff
@@ -44,6 +45,8 @@ pub mod ffi {
             ))
         }
 
+        /// Create a map for the `Script`/`Script_Extensions` properties, using compiled data.
+
```
Comment (sffc) [inline ffi/capi/src/script.rs:49]: Whitespace is still funny

**Review by sffc — COMMENTED** [review]
I annotated all the places I found with whitespace issues

**Comment by Manishearth** [comment]
Oh, you didn't have to do that, I just didn't realize this issue was in more than one file.

**Review by sffc — APPROVED** [review]

**Pull request merged**